lib/cosmos/interfaces/cmd_tlm_server_interface.rb

```diff
@@ -59,7 +59,7 @@ def read
         end
       end
 
-      while true
+      while connected?
```

Comment:
I noticed a stack trace when shutting down the server due to the CmdTlmServer interface not able to get_limits_event because the @limit_id was nil so I avoid this with connected?